### PR TITLE
Task 10,11

### PR DESCRIPTION
**Pull request by Sudheer2331**
Task 11 and 10 completed using flowchart with all the parameters

### CODE REVIEW

**Comment by vajrang-b** [comment]
SourceCode/2023/Oct2023/Basics/Sudheer Devabattula/Task010/project.json
Based on my review:

1. Main.xaml:
   - You've a sequence called "Selecting Weekdays" which seems to be redundant. In plain English, you've some code which isn't needed or it's repeating. It would be better to remove such unnecessary pieces of code to keep the workflow tidy and efficient. You can do this via the "Remove Surrounding Sequence" option in UiPath Studio.

Score: Code completion: 1.5, Execution: 2, Annotations and Comments: 0. 

2. NoFile:
   - It looks like your project doesn't have the Automation Hub URL defined, this is required by your organization. You can add this missing bit in the "Project Settings".

Score: Code completion: 1, Execution: 1, Annotations and Comments: 0.

Remember, clearer and cleaner code is easier to understand and less prone to errors.

**Comment by vajrang-b** [comment]
SourceCode/2023/Oct2023/Basics/Sudheer Devabattula/Task011/project.json
Score:

Code Completion: 1.5/2 - There are some elements missing such as the Log Message in some workflows and the Automation Hub URL that needs to be defined.

Execution: 1.5/2 - The code seems to be working but has significant room for improvement like the multiple instances of 'LogMessage' display name and some activities using default name.

Annotations and comments: 0.5/1 - There are missing log messages and some activities with default names that are too generic to understand.

Total score: 3.5/5

Code Review:

On the file "Main.xaml"
There is no log message used in workflow 'Main'. Log Messages are like signposts on the road, they guide on what has gone down at each step, and help debug when the error occurs. Add them.

On the file "User Selection.xaml",
You are using the 'LogMessage' display name repeatedly. Try using distinct, descriptive names for clarity and ease of understanding. It's okay to take a few extra seconds to frame a good descriptive name. Also, there is an activity named 'Log Message' that still has its default name. Change it to something more meaningful. 

On the file that seems to be missing or misnamed,
Ensure that

**Comment by krish952-creator** [comment]
@Sudheer2331 
Task -10
Code verified. 
Task -11
Code verified. Default activity name 
image
